generatedocs/data/config/scheduler/description.md

```diff
@@ -1,3 +1,17 @@
 ## Scheduler section
 
 This section of the JSON file contains scheduler settings for the users in the load scenario.
+
+### Using "reconnectsettings"
+
+If `reconnectsettings.reconnect` is enabled the tool will attempt the following:
+
+* Re-connect the websocket.
+* Get the currently opened app in re-attached engine session.
+* Re-subscribe to the same object has before getting disconnected.
+* Restart the action which was ongoing during the disconnect.
+* If successful the action where the re-connect happened will be logged as a successful action with changed `action` and `label`, these will instead be `Reconnect(action)` and `Reconnect(label)`.
```

Comment:
Suggestion:
"* If successful, the action during which the re-connect happened is logged as a successful action with `action` and `label` changed to `Reconnect(action)` and `Reconnect(label)`."